include/zephyr/bluetooth/assigned_numbers.h

```diff
@@ -0,0 +1,1465 @@
+/** @file
+ *  @brief Bluetooth Assigned Numbers, codes and identifiers.
+ */
+
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_ASSIGNED_NUMBERS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_ASSIGNED_NUMBERS_H_
+
+/**
+ * @brief Bluetooth Assigned Numbers, codes and identifiers.
+ * @defgroup bt_assigned_numbers Assigned Numbers.
+ * @since 1.0
+ * @version 1.0.0
+ * @ingroup bluetooth
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Company Identifiers (see Bluetooth Assigned Numbers)
+ * @{
+ */
+#define BT_COMP_ID_LF           0x05f1 /**< The Linux Foundation */
+/**
+ * @}
+ */
+
+/**
+ * @name Appearance Numbers.
+ *
+ * Last Modified on 2023-01-05
+ * @{
```

Comment:
Should there be a super group for `Core` for this? 

We could in theory have the same structure as the assigned numbers document if we wanted (but also something we can easily add later)